src/atoms_calculators.jl

```diff
@@ -0,0 +1,76 @@
+#
+# Define AtomsCalculators interface for DFTK.
+#
+# This interface is inspired by the one used in Molly.jl, 
+# see https://github.com/JuliaMolSim/Molly.jl/blob/master/src/types.jl
+#
+# TODO: Decide how Hamiltonian terms should be updated upon change of the positions.
+# These terms are held inside the `basis`. Currently, the whole basis is rebuilt.
+# TODO: Find out if symmetries can be re-enabled (see issue on GH).
+
+using AtomsBase
+using AtomsCalculators
+
+Base.@kwdef struct DFTKParameters
```

Comment:
Maybe add a super brief docstring, documenting the minimal requirements:
```suggestion
"""
Construct a [AtomsCalculators](https://github.com/JuliaMolSim/AtomsCalculators.jl) compatible calculator
for DFTK. The `model_kwargs` are passed onto the [`Model`](@ref) constructor, the `basis_kwargs` to the [`PlaneWaveBasis`](@ref) constructor, the `scf_kwargs` to [`self_consistent_field`](@ref). At the very least the DFT `functionals` and the `Ecut` needs to be specified.

## Example
```julia-repl
julia> DFTKParameters(; model_kwargs=(; functionals=[:lda_x, :lda_c_vwn]),
                       basis_kwargs=(; Ecut=10, kgrid=(2, 2, 2)),
                       scf_kwargs=(; tol=1e-4))
```
"""
```


(the formatting is messed up ... look at the code and copy n' paste what is between the `suggestion` codeblock.)